packages/component-library/src/components/plmg-radio-button/plmg-radio-button.scss

```
@@ -0,0 +1,127 @@
+@use '~@ducky/plumage-tokens/dist/scss/default' as tokens;
+
+// Component style
+// -------------------------------------------------
+.radio-button-wrapper {
+  display: flex;
+  cursor: pointer;
+  align-items: center;
+  width: fit-content;
+  border-radius: tokens.$plmg-border-radius-s;
+
+  &:hover {
+    input[type='radio'] {
+      background-color: tokens.$plmg-color-background-primary-strong;
+      border-color: tokens.$plmg-color-border-primary-strong;
+      transition: background-color 0.2s ease, border-color 0.2s ease;
+
+      &:checked {
+        background-color: tokens.$plmg-color-background-neutral-hover;
+        border-color: tokens.$plmg-color-border-primary-strong;
+        & ~ label {
+          color: tokens.$plmg-color-text-neutral;
+        }
+      }
+    }
+
+    input[type='radio']:checked::after {
+      box-shadow: inset 1em 1em
+        tokens.$plmg-color-background-primary-strong-hover;
+    }
+
+    label {
+      color: tokens.$plmg-color-text-primary-strong;
+    }
+  }
+
+  &:active {
+    input[type='radio']:checked::after {
+      box-shadow: inset 1em 1em
+        tokens.$plmg-color-background-primary-strong-active;
+    }
+  }
+
+  &:focus-within {
+    outline: tokens.$plmg-border-width-l solid
+      tokens.$plmg-color-border-standout;
+  }
+}
+
+input[type='radio'] {
```

By specifying this outside any class and having this component non "shadow", you are actually applying this style to **every radio button on the page**, potentially messing up with other styles and other libraries.

You need to encapsulate this into your `.radio-button-wrapper` class or use a class and apply it to the input.

---

For the other component, we've created a class and applied it to the component.
 In your case, you would name it `plmg-radio-button`.
And in the `.tsx`, do something like `const inputClasses = {'plmg-radio-button': true, ...`

```suggestion
.plmg-radio-button {
```